cpu/stm32l1/periph/i2c.c

```diff
@@ -116,9 +116,9 @@ int i2c_init_master(i2c_t dev, i2c_speed_t speed)
         /* disable peripheral */
         i2c->CR1 &= ~I2C_CR1_PE;
         /* toggle both pins to reset analog filter */
-        _toggle_pins(port, pin_scl, pin_sda);
+        _toggle_pins(dev);
```

Comment:
@ReneHerthel in line 113 an errata sheet was mentioned. I **assume** the whole block was copied from one CPU once and dragged along even if the errata doesn't apply to all stm32 CPUs. Can you please find out if the errata sheet applies to this CPU and if not, remove the whole `if (i2c->SR2 & I2C_SR2_BUSY)` block and test if driver still works?